appserver/web/web-glue/src/main/java/com/sun/enterprise/web/PEAccessLogValve.java

```diff
@@ -908,6 +908,14 @@ void updateAccessLogAttributes(HttpService httpService,
             historyFiles = new LinkedList<File>();
         }
         
+        //Conditional access log
+        if (accessLogConfig != null) {
```

Comment:
Why check if not null and then do an else that sets it to null. Why not just set it regardless and not bother with the getDefaultCondition change?